lib/new_relic/agent/connect/request_builder.rb

```diff
@@ -24,7 +24,7 @@ def connect_payload
             :host => local_host,
             :display_host => Agent.config[:'process_host.display_name'],
             :app_name => Agent.config[:app_name],
-            :language => 'ruby',
+            :language => LANGUAGE,
```

Comment:
If the capitalization isn't important, in the future, this could be combined with the `Ruby` constant we're adding for AIM's ingest source.

Reply:
My current hypothesis is that the string value is currently only expected to be read by humans so having a capital letter ought to be ok, but I'd like to be consistent with the example provided by the agent spec and the existing agents that are going with all lowercase.